src/vs/workbench/contrib/terminal/browser/terminalTabsList.ts

```diff
@@ -735,6 +735,8 @@ class TerminalTabsDragAndDrop implements IListDragAndDrop<ITerminalInstance> {
 		instance.focus();
 		await instance.sendPath(resource, false);
 	}
+
+	dispose(): void { }
```

Comment:
I think you'll have conflicts with my recently merged pr https://github.com/microsoft/vscode/pull/193666

Reply:
Don't see GH complaining about any conflicts.

Reply:
Sneaky conflicts 😉